crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/reports/scheduler/ReactivatedProsecutionCaseReportScheduler.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.laa.crime.crowncourt.reports.scheduler;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import uk.gov.justice.laa.crime.crowncourt.reports.service.ReactivatedProsecutionCaseReportService;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.io.IOException;
+
+@Getter
```

Comment:
I don't think we need the `@Getter` annotation on this class.

Reply:
done